wordpress-mobile/WordPress-Editor-iOS
# The ZSSField callbacker does not directly reference the editor's callback method anymore.

This is the first PR in a series of upcoming PRs to abstract `ZSSField` from `ZSSEditor`.  The idea behind these changes it to make the code more modular and eventually easier to maintain and unit-test (yep, planning to add unit-tests for our editor!).

`ZSSField` was executing callbacks by directly referencing `ZSSEditor.callback()`.  This was bad design-wise because it meant the classes were highly tied to one another (thus making it very hard to instantiate `ZSSField` without `ZSSEditor` to unit test it).

**How to test:**
- Increate the app's logging level to ALL in `WPEditorLoggingConfiguration.m`.
- Run the app.
- Edit the fields, change selections, etc.
- Make sure field-delivered callbacks are still received by the app.
